### ISSUE TITLE
Empty Data tab

### ISSUE THREAD

**Issue by Globegitter** [emberjs/ember-inspector#319]
I am using the following versions:
```
Ember-cli: 0.2.0-beta.1
Ember Inspector 1.7.2
Ember 1.12.0-beta.1+canary.f5873415
Ember Data 1.0.0-beta.15
jQuery 2.1.3
```
with POD Structure and loading in my `dashboard/route.js` the following model:
```js
model: function() {
    return this.store.find('researchProject')
  }
```

This loads into my controller as expected and also shows up fine under the resolved Promises but my Data Tab is completely empty not showing anything. Any idea why this could be?

Let me know if you need any more details/information.

**Comment by teddyzeenny** [comment]
If your models are also using POD structure (`research-project/model.js`), you need to update `bower.json` to use `ember-resolver` master branch as it hasn't been released yet. See https://github.com/emberjs/ember-inspector/issues/298#issuecomment-72726462

**Comment by Globegitter** [comment]
Yep that fixed it - thanks.

**Comment by stefanpenner** [comment]
@teddyzeenny can you open a resolver issue. To remind me to  release when I get home

**Comment by rwjblue** [comment]
I tagged/released 0.1.12 of ember-resolver.
